eos/statistics/log-prior.cc

```diff
@@ -753,6 +756,259 @@ namespace eos
                     return true;
                 }
         };
+
+        /*!
+         * Transformed and shifted multivariate uniform prior distribution
+         */
+        class Transform :
+            public LogPrior
+        {
+            private:
+                std::vector<Parameter> _parameters;
+
+                std::vector<QualifiedName> _names;
+
+                // inputs
+                gsl_vector * const _shift;
+                gsl_matrix * const _transform;
+                gsl_vector * const _min;
+                gsl_vector * const _max;
+
+                // the normalization constant of the density
+                double _value;
+
+                // cholesky matrix of covariance, inverse of the cholesky matrix, and inverse of covariance
```

Comment:
This comment is outdated. It lists 3 matrices, but only 2 matrix pointers are declared below.